script/preview.js

```diff
@@ -62,6 +63,17 @@ app.get('/preview', async (req, res) => {
       ...drupalPage,
       isPreview: true,
       isDrupalPage: true,
+      headerFooterData: JSON.parse(
+        fs.readFileSync(
+          path.join(
+            __dirname,
+            '..',
+            options.buildpath,
+            'generated/headerFooter.json',
```

Comment:
Should this be `drupalHeaderFooter.json`?

Reply:
Ideally, yes. However, I left it like this because I wasn't sure if the drupal file would exist in the tarball we're pulling down, until the deploy tomorrow. So this was just to give us more testing flexibility. I'm planning on pointing it to the drupal version once we get things working.

Reply:
That makes sense, thanks @jbalboni!